dev/MRTCore/mrt/mrm/include/mrm/build/Atoms.h

```diff
@@ -22,21 +22,21 @@ class FileAtomPoolBuilder : public ISectionBuilder, public IAtomPoolWriter
         fStringPoolIsOwned = 0x0100
     };
 
-    bool m_finalized;
+    bool m_finalized{ false };
 
-    UINT32 m_flags;
-    Atom::HashMethod m_hashMethod;
+    UINT32 m_flags{ 0 };
+    Atom::HashMethod m_hashMethod{ DEF_HASH_DEFAULT };
 
-    AtomPoolGroup* m_group;
-    Atom::PoolIndex m_poolIndex;
+    AtomPoolGroup* m_group{ nullptr };
+    Atom::PoolIndex m_poolIndex { DEF_ATOM_POOL_INDEX_NONE };
 
-    Atom::AtomCount m_numAtoms;
-    Atom::AtomCount m_sizeAtoms;
-    DEFFILE_ATOMPOOL_HASHINDEX* m_hash;
-    UINT32* m_offset;
-    WriteableStringPool* m_pStrings;
-    WCHAR m_description[FileAtomPool::DescriptionLength];
-    BaseFile::SectionIndex m_sectionIndex;
+    Atom::AtomCount m_numAtoms{ DEF_ATOM_MAX_COUNT_SMALL };
+    Atom::AtomCount m_sizeAtoms{ DEF_ATOM_MAX_COUNT_SMALL };
```

Comment:
0?

Reply:
Yea wasn't sure about this one so needed your feedback. I think zero would be better.